src/curve/LinearIncreasingCurve.sol

```diff
@@ -101,7 +99,13 @@ contract LinearIncreasingCurve is
     //////////////////////////////////////////////////////////////*/
 
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
+    constructor(int256[3] memory _coefficients, uint256 _maxEpochs) {
+        SHARED_CONSTANT_COEFFICIENT = _coefficients[0];
+        SHARED_LINEAR_COEFFICIENT = _coefficients[1];
+        SHARED_QUADRATIC_COEFFICIENT = _coefficients[2];
```

Comment:
Because Quadratic is incompatible w. current version of contracts should revert if >0